dot/network/host.go

```diff
@@ -276,32 +286,10 @@ func (h *host) writeToStream(s libp2pnetwork.Stream, msg Message) error {
 	return err
 }
 
-// broadcast sends a message to each connected peer
-func (h *host) broadcast(msg Message) {
-	for _, p := range h.peers() {
-		err := h.send(p, "", msg)
-		if err != nil {
-			logger.Error("Failed to broadcast message to peer", "peer", p, "error", err)
-		}
-	}
-}
-
-// broadcastExcluding sends a message to each connected peer except specified peer
-func (h *host) broadcastExcluding(msg Message, peer peer.ID) { //nolint
-	for _, p := range h.peers() {
-		if p != peer {
-			err := h.send(p, "", msg)
-			if err != nil {
-				logger.Error("Failed to send message during broadcast", "peer", p, "err", err)
-			}
-		}
-	}
-}
-
```

Comment:
I'm assuming these are being removed because they're not needed, is there no longer a need to broadcast messages?

Reply:
yes, they weren't actually being used anywhere. notifications messages are still being "broadcast" (gossiped) in the `broadcastExcluding` function in notifications.go